### ISSUE TITLE
Importing bug

### ISSUE THREAD

**Issue by mattjared** [xxhomey19/nba-color#13]
Using the latest version I'm running into an error trying to import the package. Given the following component in a react app (generated with create-react-app) I'm getting a rambda error in the console when I attempt to view the app. 

Code: 
```javascript
import React, { Component } from 'react';
import { getMainColor } from 'nba-color';
class Header extends Component {
  render() {
    const color = getMainColor('LAL');
    return (
      <div className="Header">{ color }</div>
    );
  }
}
export default Header;
```

Error in the console preventing the page from loading...
```bash
Failed to compile.
./node_modules/nba-color/lib/nba-color.mjs
Attempted import error: 'ramda' does not contain a default export (imported as 'e').
```

Any idea what might be causing this?

**Comment by xxhomey19** [comment]
Sorry it's my fault. Will fix it asap. Thanks for reporting the issue.

**Comment by xxhomey19** [comment]
Hey @mattjared 
This issue should be fixed by 312016f80da5e01b30e54779f961dac503c71c41
Please let me know whether it works for you thanks.

**Comment by mattjared** [comment]
Thanks! Working flawlessly now.
